app/assets/javascripts/__tests__/search.test.js

```diff
@@ -0,0 +1,161 @@
+'use strict'
+
+// set the window.jQuery variable that the file expects
+window.jQuery = require('../../../../vendor/assets/javascripts/product-page-template/vendor/jquery.js')
```

Comment:
I'm not quite sure how we solve this, but this isn't actually the version of jQuery we're using, we're using the one provided by the `jquery-rails` gem. To ensure we're testing against the correct version, we should probably work out how to use that one.